example/tests/test_helpers.py

```diff
@@ -46,6 +48,20 @@ def __str__(self):
         return 'Related object'
 
 
+class ArticleProxyModel(Article):
```

Comment:
I am not sure how is this ad-hoc model testing typically done. This approached worked here so I used it. I am welcome to be guided for best practices here.

Reply:
Look OK to me.